source/platforms/github/_tests/_github_api.test.ts

```diff
@@ -363,9 +363,26 @@ describe("Peril", () => {
       delete process.env.PERIL_BOT_USER_ID
     })
 
-    it("Makes getUserId return undefined", async () => {
+    it("Makes getUserId return PERIL_BOT_USER_ID", async () => {
       const userID = await api.getUserID()
       expect(userID).toBe(1)
     })
   })
+
+  describe("Allows setting GHE_ACTIONS_BOT_USER_ID env variable", () => {
```

Comment:
`GHE_ACTIONS_BOT_USER_ID` — this is a custom environment variable, right? (not set by github, set by the user).

The naming convention for DangerJS custom environment variables is to prefix it with `DANGER_`, so in this case, I would name it something like `DANGER_GHE_ACTIONS_BOT_USER_ID`

Reply:
Also, of course, once we need to document this env-var in appropriate places. Probably with a section for “GitHub Enterprise”

Reply:
Yep makes sense. I have update the name of the env variable and added some docs. Please let me know if I missed something.